src/main/java/com/minecolonies/coremod/blocks/huts/BlockHutEnchanter.java

```diff
@@ -0,0 +1,34 @@
+package com.minecolonies.coremod.blocks.huts;
+
+import com.minecolonies.api.blocks.AbstractBlockHut;
+import com.minecolonies.api.colony.buildings.ModBuildings;
+import com.minecolonies.api.colony.buildings.registry.BuildingEntry;
+import com.minecolonies.api.tileentities.TileEntityEnchanter;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.World;
+import org.jetbrains.annotations.NotNull;
+
+public class BlockHutEnchanter extends AbstractBlockHut<BlockHutEnchanter>
+{
+    @NotNull
+    @Override
+    public String getName()
+    {
+        return "blockhutenchanter";
```

Comment:
Name doesn’t Match capitalisation style of other huts

Reply:
on purpose, we should make new ones more often like this, so its compat with 1.14 too